README.md

```diff
@@ -15,7 +15,10 @@ Theory of static single assignment developed in the Lean proof assistant.
 We also build a wealth of tooling to interact closely with the [MLIR compiler ecosystem](https://mlir.llvm.org/),
 to enable workflows that include formal verification in the day-to-day of MLIR development.
 
-- [Link to generated documentation](https://opencompl.github.io/lean-mlir/)
+#### Documentation
+
+- **Publication: [Verifying Peephole Rewriting In SSA Compiler IRs](https://arxiv.org/abs/2407.03685)**
+- [API Documentation (auto-generated)](https://opencompl.github.io/lean-mlir/)
```

Comment:
You seem to be on an older branch. Can you `git merge origin/main` to remove these differences?